### ISSUE TITLE
Header Accessibility

### ISSUE THREAD

**Issue by moisescodes** [moisescodes/moisescodes.github.io#1]
The text elements in the page header are just under the WCAG accessibility guidelines. Contrast should be increased by adjusting text wight or size.
